MapboxNavigation/NavigationComponent.swift

```diff
@@ -2,4 +2,7 @@ import Foundation
 import MapboxCoreNavigation
 import CoreLocation
 
-@objc protocol NavigationComponent: NavigationServiceDelegate {}
+/*
+ A NavigationComponent is any component that can receive NavigationService messages.
```

Comment:
What is a “NavigationService message”, in terms that a developer who’s unaware of this library’s internals can understand?